bitaxe_hashrate_benchmark.py

```
@@ -304,242 +556,606 @@ def benchmark_iteration(core_voltage, frequency):
-            reset_to_best_setting()
-            save_results()
-            print(GREEN + "Bitaxe reset to best settings and results saved." + RESET)
+        if benchmark_mode == "refine":
+            run_refine()
+        elif benchmark_mode == "efficiency":
+            run_efficiency()
         else:
-            print(YELLOW + "No valid benchmarking results found. Applying predefined default settings." + RESET)
-            set_system_settings(default_voltage, default_frequency)
+            run_grid()
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)